unitTests/ui/FacetValueSuggestionsTest.ts

```diff
@@ -368,21 +372,47 @@ export function FacetValueSuggestionsTest() {
       });
     });
 
-    describe('when only using the search box keywords and the keyword is empty', () => {
+    describe('when only using the search box keywords', () => {
       beforeEach(() => {
         test.cmp.options.useValueFromSearchbox = true;
         test.cmp.options.useQuerySuggestions = false;
-        setUpKeywordInOmnibox('');
       });
 
-      it('returns no suggestions', async done => {
+      it(`returns no suggestions`, async done => {
+        setUpKeywordTextInOmnibox(anOmniboxSuggestionKeyword.text);
+        const resultingArgs = await triggerPopulateOmniboxEvent();
+
+        firstSuggestion(resultingArgs).then(result => {
```

Comment:
You could use `await` here. Also, when there is an `async` keyword before the handler (the second arg of the `it` function), there is no need to use `done`. Jasmine will know the test is async.

https://jasmine.github.io/tutorials/async

Reply:
I noticed that but there was a lot of tests done this way and I simply copy and pasted the same thing for consistency. I'll change them all to use `await` instead.

As for removing `done`, it looks like Jasmine is not awaiting promises despite the documentation saying it does, removing `done` results in multiple tests running in parallel and giving errors.